### DISCUSSION TITLE
TorchGeo could really benefit from a SuperResolutionTask and a PansharpeningTask

### DISCUSSION THREAD

**Discussion by kaushikCanada** [torchgeo/torchgeo#1913] [Ideas]
The community have been working with deep learning superresolution and pansharpening and there are datasets like Pancollection and pairmax. A whole community of people is missing out on torchgeo and has to write custom code (like me :)). Maybe this is a useful suggestion.

**Comment by adamjstewart** [comment]
Would love to see one. Want to try contributing one? https://torchgeo.readthedocs.io/en/stable/user/contributing.html
